corehq/form_processor/signals.py

```diff
@@ -0,0 +1,4 @@
+from django.dispatch.dispatcher import Signal
+
+
+sql_case_post_save = Signal(providing_args=["case"])
```

Comment:
yeah that's the end state i was thinking of, but the way i saw this working in the meantime is we'd temporarily add this new signal until all of the post-save signals are sql compatible, and then we'd just drop the extra signal and only use one